base/src/main/scala/Prelude.scala

```diff
@@ -55,13 +61,23 @@ trait Prelude  extends data.DisjunctionFunctions
   implicit def PtraversableOps[T[_], A](ta: T[A])(implicit T: Traversable[T]): TraversableSyntax.Ops[T, A] =
     new TraversableSyntax.Ops(ta)
 
+  implicit def PProfunctorOps[P[_,_], A, B](pab: P[A, B])(implicit P: Profunctor[P]): ProfunctorSyntax.Ops[P, A, B] = 
+    new ProfunctorSyntax.Ops(pab)
+  
+  implicit def PChoiceOps[P[_,_], A, B](pab: P[A, B])(implicit P: Choice[P]): ChoiceSyntax.Ops[P, A, B] =
+   new ChoiceSyntax.Ops(pab)
+
+  implicit def PStrongOps[P[_,_], A, B](pab: P[A, B])(implicit P: Strong[P]): StrongSyntax.Ops[P, A, B] =
+    new StrongSyntax.Ops(pab)
```

Comment:
The naming convention set above seemed to be `PprofunctorOps`, `PchoiceOps`, `PstrongOps`.